### ISSUE TITLE
Fail to reproduce NYU dataset accuracy

### ISSUE THREAD

**Issue by kebijuelun** [astra-vision/MonoScene#41]
Thanks for this great work,  I found the nyu accuracy is hard to reproduce,
the accuracy with my self-trained model:
```
Precision=56.4726, Recall=61.7487, IoU=41.8369                                                                                                                                      
class IoU: ['empty', 'ceiling', 'floor', 'wall', 'window', 'chair', 'bed', 'sofa', 'table', 'tvs', 'furn', 'objs'],                                                                 
88.7854,  7.9803,  93.3229,  11.3533,  9.9692,  14.0669,  45.8456,  35.6397,  14.4716,  9.2469,  27.8702,  11.9180,                                                                 
mIoU=25.6077                                                                                                                                                                        
--------------------------------------------------------------------------------                                                                                                    
DATALOADER:0 TEST RESULTS                                                                                                                                                           
{'test/loss': 6.906957626342773,                                                                                                                                                    
 'test/loss_geo_scal': 1.1675410270690918,                                                                                                                                          
 'test/loss_relation_ce_super': 0.6730754375457764,                                                                                                                                 
 'test/loss_sem_scal': 2.944209098815918,                                                                                                                                           
 'test/loss_ssc': 2.1221349239349365} 
```

the results in the paper:

![image](https://user-images.githubusercontent.com/29164878/209455709-1a3999c4-b26d-4af1-8c0d-35e0989fa6fe.png)

- my reproduce process is same as readme 
- reproduce machine: 2xV100

BTW, the kitti reproduce accuracy is also lower than the paper and release model:

| experiments | SC IoU | mIoU| Precision | Recall | 
--- | --- | --- | --- | --- | 
results from paper | 37.12 | 11.50 | \ | \ 
results from github release model |  36.7950 | 11.2993 | 52.1908 | 55.5026 
reproduce-2xV100 | 36.1542 |10.7542  | 50.7314 | 55.7177 | 
reproduce-4xA100 | 36.4819 |10.8438  | 50.1127 | 57.2874 |

Is the difference caused by the validation set and test set? or the difference is a normal accuracy jitter？
Do you have any comments or suggestions about this problem？

Thanks.

**Comment by anhquancao** [comment]
Hi @kebijuelun, the performance that you got is from the best checkpoint or the last checkpoint?

**Comment by kebijuelun** [comment]
> Hi @kebijuelun, the performance that you got is from the best checkpoint or the last checkpoint?

My repoduce cmd follows the readme exactly(enable_log=true for kitti, enable_log=false for NYU). So the performance for kitti is the best checkpoint (with mIoU) , and the performance for NYU is the last checkpoint.

And I also tried to set enable_log=true with NYU experiment, the best ckpt accuracy will be:
```
Precision=54.5225, Recall=65.7073, IoU=42.4447
class IoU: ['empty', 'ceiling', 'floor', 'wall', 'window', 'chair', 'bed', 'sofa', 'table', 'tvs', 'furn', 'objs'], 
88.2998,  9.2492,  93.3849,  12.0239,  11.7991,  14.4482,  46.8162,  37.4417,  15.5499,  13.5113,  28.4804,  12.8191, 
mIoU=26.8658
```

the performance with the best epoch can basically achieve the results of the paper.

**Comment by anhquancao** [comment]
Thanks for your information! Let me check and will get back to you. Probably I accidentally change some settings when cleaning the code for releasing.

**Comment by anhquancao** [comment]
For NYUv2, I think we reported the best epoch following other methods in the literature. For SemKitti, I will need to rerun the code to check.
